### PR TITLE
XIVY-12892 release to market via pipeline

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

**Review by ivy-cst — APPROVED** [review]
Looks good :+1: 

In the end you should look that the version number is also updated on the version raise scripts

**Comment by ivy-lmu** [comment]
> Looks good 👍
> 
> In the end you should look that the version number is also updated on the version raise scripts

thanks, I have changed it to `${npm_package_version:0:4}`

File: extension/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "lint": "eslint --ext .ts,.tsx ./src",
     "package": "vsce package --yarn",
     "package:with:engine": "yarn package -o ${npm_package_name}-with-engine-${npm_package_version}.vsix",
```
Comment (ivy-cst) [inline extension/package.json:24]: May you have also do adjust this one?

Comment (ivy-cst) [reply extension/package.json:24]: Yes, true. But may be you could still change the version inside in the pom? So we see the exact version if we install it this way?

Comment (ivy-lmu) [reply extension/package.json]: changed that, now the output is designer-11.vsix and inside pom the version is 11.3.2023xxxx

**Pull request merged**